app/src/main/java/com/github/dedis/student20_pop/ui/event/AbstractEventFragment.java

```diff
@@ -0,0 +1,215 @@
+package com.github.dedis.student20_pop.ui.event;
+
+import android.app.Activity;
+import android.content.Intent;
+import android.text.InputType;
+import android.text.TextWatcher;
+import android.view.View;
+import android.widget.EditText;
+import android.widget.Toast;
+
+import androidx.appcompat.app.AppCompatDialogFragment;
+import androidx.fragment.app.Fragment;
+import androidx.fragment.app.FragmentManager;
+
+import com.github.dedis.student20_pop.R;
+import com.github.dedis.student20_pop.ui.DatePickerFragment;
+import com.github.dedis.student20_pop.ui.TimePickerFragment;
+
+import java.text.DateFormat;
+import java.text.ParseException;
+import java.text.SimpleDateFormat;
+import java.util.Date;
+import java.util.Locale;
+
+/**
+ * Multiples Event Creation Fragment have in common
+ * that they implement start/end date and start/end time.
+ *
+ * This class handles these fields.
+ */
+abstract class AbstractEventFragment extends Fragment {
```

Comment:
Good idea! maybe just specify that this is for when we create events.

Reply:
I'll rename it